hackthon issue 799:Change OpenFL documentation font to improve accessibility

wangleflex opened the pull request:
This PR is for replacing the font of OpenFL documents with Intel One Mono font for low vision developers.
IntelOne Mono font(for WEB) is used in customized CSS file "docs/_static/css/Intel_One_Mono_Font_Theme.css"

Known issues:
1. The text font within the images has not been replaced.
2. Some icons that do not exist in the new font has not been replaced(home icon, circle-arrow .etc.).
 Fixes #799

psfoley commented:
Thanks for the contribution, @wangleflex and team! Could you please retroactively sign the commit per these instructions: https://github.com/securefederatedai/openfl/pull/809/checks?check_run_id=13058829834

danhe1 committed:
Change OpenFL documentation font to improve accessibility
    This replaces the font of OpenFL documents with Intel One Mono font for low vision developers.

    Known issues:
    1. The text font within the images has not been changed.
    2. Some icons that do not exist in the new font cannot be displayed properly.

    Fixes securefederatedai#799

    Co-authored-by: Wang, Le <le3.wang@intel.com>
    Signed-off-by: He, Dan H <dan.h.he@intel.com>
    Signed-off-by: Jiang, Jiaqiu <jiaqiu.jiang@intel.com>
    Signed-off-by: Li, Qingqing <qingqing.li@intel.com>
    Signed-off-by: Wang, Le <le3.wang@intel.com>
    Signed-off-by: Wu, Caili <caili.wu@intel.com>

Signed-off-by: He, Dan H <dan.h.he@intel.com>
